dizoo/classic_control/mountain_car/config/mtcar_rainbow_config.py

```diff
@@ -0,0 +1,58 @@
+from easydict import EasyDict
+
+# DI-Engine uses EasyDict for configuration, by convention
+mtcar_rainbow_config = EasyDict(dict(
+    exp_name='mtcar_rainbow_seed0',
+    env=dict(
+        collector_env_num=8,
+        evaluator_env_num=5,
+        n_evaluator_episode=5,
+        stop_value=195,
+    ),
+    policy=dict(
+        cuda=False,
+        priority=True,
+        discount_factor=0.97,
```

Comment:
there are some fields with high priority to tune final performance:
- discount_factor: [0.97, 0.99, 0.995]
- nstep: [1, 3, 5]
- learning_rate: [1e-3, 3e-4, 1e-4] or more complex lr scheduler scheme
- update per collect: [3, 5, 10]
- buffer size: [2w, 5w, 10w]

Reply:
Config file will be updates on branch "mtcar_rainbow" in #471. Will reflect these notes in #471 as well.